docs/content/en/docs/tasks/cluster/cluster-upgrades.md

```diff
@@ -36,15 +36,53 @@ EKS Anywhere `upgrade` also supports upgrading the following core components:
 * Cert-manager
 * Etcdadm CAPI provider
 * EKS Anywhere controllers and CRDs
-* Gitops controller (Flux) - this is an optional component, will be upgraded only if specified
+* GitOps controllers (Flux) - this is an optional component, will be upgraded only if specified
 
 The latest versions of these core EKS Anywhere components are embedded into a bundles manifest that the CLI uses to fetch the latest versions 
 and image builds needed for each component upgrade. 
 The command detects both component version changes and new builds of the same versioned component.
 If there is a new Kubernetes version that is going to get rolled out, the core components get upgraded before the Kubernetes
 version. 
 Irrespective of a Kubernetes version change, the upgrade command will always upgrade the internal EKS
-Anywhere components mentioned above to their latest available versions. These upgrade changes are backwards compatible.
+Anywhere components mentioned above to their latest available versions. Most upgrade changes are backwards compatible.
+
+#### GitOps controller upgrade
+
+In order to accommodate the management cluster feature, the CLI structures the repo directory following the convention:
+
+```
+clusters
+└── management-cluster
+    ├── flux-system
+    │   └── ...
+    ├── management-cluster
+    │   └── eksa-system
+    │       └── eksa-cluster.yaml
+    ├── workload-cluster-1
+    │   └── eksa-system
+    │       └── eksa-cluster.yaml
+    └── workload-cluster-2
+        └── eksa-system
+            └── eksa-cluster.yaml
+```
+*By default, Flux kustomization reconciles at the management cluster's root level (`./clusters/management-cluster`), so both management and all its workload clusters are synced.*
+
+To upgrade an existing GitOps enabled cluster that was created with older release (v0.5.0 and below), a manual Git repo update is needed before running the upgrade command. User needs to bring the eksa cluster config into a subfolder under the management cluster's root level (shown as above). And update the `clusterConfigPath` field in the `eksa-cluster.yaml`, to reflect the new path that holds the `eksa-system` directory. e.g.
```

Comment:
I think the first information we present after ["GitOps Controller Upgrade"](https://github.com/aws/eks-anywhere/pull/578/files#diff-34ef9f9e4b2404d53a4b13741331d54859a0195da2bdca57ae73e83d3626aa06R49) should be a bolded or note-boxed call-out which draws the users attention to the fact that we need to perform a manual action before upgrading; I think this should be before displaying the new convention and explaining how to change  the dir structure for existing repos

Reply:
```suggestion
__Upgrading an existing GitOps enabled cluster created by an older release of the EKS-A CLI (v0.5.0 and below) requires a manual update to the git repository structure.__ 

Before running the `upgrade` command on an existing GitOps enabled cluster, you must:
- move the eksa cluster config into a subdirectory under the management cluster's root level (as shown above).
- update the `clusterConfigPath` field of the `GitOpsConfig` of the `eksa-cluster.yaml` to reflect the new path of the `eksa-system` directory. For example:
```

Reply:
jesus only by comparison do i realize how bad my writing is. yours so short but clear